chapter_16_javascript.asciidoc

```diff
@@ -111,6 +116,7 @@ _validation.ItemValidationTest.test_error_messages_are_cleared_on_input)
 AssertionError: True is not false
 ----
 
+// RITA: Move on to do what?
```

Comment:
if you really want!  but, i feel that people kind of know what's next by now?  we've been doing the same rhythm for 16 chapters now, Ft, unit tests, code, red-green-refactor, all that stuff.